pkg/minikube/perf/binary.go

```diff
@@ -98,11 +141,9 @@ func (b *Binary) download() error {
 
 // newBinaryFromPR downloads the minikube binary built for the pr by Jenkins from GCS
 func newBinaryFromPR(pr string) (*Binary, error) {
-	pr = strings.TrimPrefix(pr, prPrefix)
-	// try to convert to int
-	i, err := strconv.Atoi(pr)
+	i, err := extractPRIDFromPath(pr)
```

Comment:
We shouldn't need to extract the PR ID from path, since it is passed into the function as a string